lib/flo/state.rb

```diff
@@ -1,5 +1,4 @@
-# Copyright © 2017, Salesforce.com, Inc.
-# All Rights Reserved.
+# Copyright © 2019, Salesforce.com, Inc. All Rights Reserved.
```

Comment:
Any reason you removed the line break here?

Reply:
None. This one's bizarre - bizarre that the automated search and replace caused this one file to lose a newline and bizarre that I didn't visually notice it in the PR. It's definitely a mistake.

Reply:
Restored newline.